cmd/ndm_daemonset/controller/device.go

```diff
@@ -0,0 +1,208 @@
+/*
+Copyright 2019 OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controller
+
+import (
+	apis "github.com/openebs/node-disk-manager/pkg/apis/openebs/v1alpha1"
+	//"github.com/operator-framework/operator-sdk/pkg/k8sutil"
```

Comment:
what does the above line mean? Is it an unused import?

Reply:
Removed.